<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceOrder</class>
 <widget class="QMainWindow" name="PlaceOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlaceOrder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(243, 248, 254)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="subitOrder">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>510</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 170, 255);
</string>
    </property>
    <property name="text">
     <string>提交订单</string>
    </property>
   </widget>
   <widget class="QLabel" name="fromWhere_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="choosePassenger">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 170, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>+ 选择乘机人</string>
    </property>
   </widget>
   <widget class="QLabel" name="arrow">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;&gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="spentTime">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string>33小时56分</string>
    </property>
   </widget>
   <widget class="QLabel" name="toWhere_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="firstClassChoose">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string>头等舱 ￥300</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>91</y>
      <width>431</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;  
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="arrivalTime_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="confirmLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string>确认订单</string>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>12月17日</string>
    </property>
   </widget>
   <widget class="QLabel" name="flightNumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="week">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>周二</string>
    </property>
   </widget>
   <widget class="QLabel" name="beginTime_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="businessClassChoose">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string>商务舱 ￥200</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="economyClassChoose">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string>经济舱 ￥100</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView::section{
background:rgb(255, 255, 255);
	color: rgb(0, 170, 255);
	border: none;
}
QScrollBar{background:transparent; height:12px;}
	QScrollBar::handle{background:rgb(224, 248, 255); border:2px solid transparent; border-radius:5px;}
	QScrollBar::handle:hover{background:gray;}
	QScrollBar::sub-line{background:transparent;}
	QScrollBar::add-line{background:transparent;}
QTableWidget::item { background-color: rgb(244, 244, 244); }
QTableWidget::item:nth-child(odd) { 
	background-color: rgb(231, 255, 255); }

QTableWidget { border: none; }</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>是否选中</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>电话号码</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="choosePassenger_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 170, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>- 删除乘机人</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_num">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fromWhere_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="toWhere_8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>subitOrder</zorder>
   <zorder>fromWhere_7</zorder>
   <zorder>choosePassenger</zorder>
   <zorder>arrow</zorder>
   <zorder>spentTime</zorder>
   <zorder>toWhere_7</zorder>
   <zorder>firstClassChoose</zorder>
   <zorder>arrivalTime_7</zorder>
   <zorder>confirmLabel</zorder>
   <zorder>date</zorder>
   <zorder>flightNumber</zorder>
   <zorder>week</zorder>
   <zorder>beginTime_7</zorder>
   <zorder>businessClassChoose</zorder>
   <zorder>economyClassChoose</zorder>
   <zorder>tableWidget</zorder>
   <zorder>choosePassenger_2</zorder>
   <zorder>lbl_num</zorder>
   <zorder>fromWhere_8</zorder>
   <zorder>toWhere_8</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
